cadence/contracts/bridge/FlowEVMBridge.cdc

```diff
@@ -221,13 +204,6 @@ contract FlowEVMBridge {
             )
             assert(callResult.status == EVM.Status.successful, message: "Tranfer to bridge recipient failed")
         }
-        emit BridgedNFTToEVM(
-            type: tokenType,
-            id: tokenID,
-            evmID: evmID,
-            to: FlowEVMBridgeUtils.getEVMAddressAsHexString(address: to),
-            evmContractAddress: FlowEVMBridgeUtils.getEVMAddressAsHexString(address:associatedAddress)
-        )
     }
 
     /// Public entrypoint to bridge NFTs from EVM to Cadence
```

Comment:
You have `calldata` listed as a parameter here, but I don't see it in the actual function definition.